source/index.txt

```diff
@@ -53,12 +53,15 @@ describe possible configurations for a BI system.
 Hosted Database and |bi-short|
 ------------------------------
 
-`MongoDB Atlas <https://www.mongodb.com/cloud/atlas>`_ provides a fully
-hosted solution for both MongoDB and |bi-short|. Atlas dedicated
-clusters (`size M10+
-<https://docs.atlas.mongodb.com/create-new-cluster/>`_)
-offer an `integrated BI Connector instance
-<https://docs.atlas.mongodb.com/bi-connection>`_ as an option.
+.. note::
+
+   If you don’t want to manage and administer the BI Connector, you can use
+   MongoDB Atlas, our fully-managed database-as-a-service. Databases hosted on
+   Atlas natively support the MongoDB Connector for BI and make it easy to
```

Comment:
s / MongoDB Connector for BI / |bi|

Reply:
Oh neat. Good to know! Where can I see a list of all of those?

Reply:
`conf.py`